build_projects/dotnet-cli-build/NuGetUtil.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.IO;
```

Comment:
The reason this is in `dotnet-cli-build` is because it needs to reference "NuGet.CommandLine.XPlat.Program", which is targeting `netcoreapp1.0`.

If I put it in the shared project, I would need to move the shared project to target `netcoreapp1.0`. Do you think that is acceptable?